reana_db/models.py

```diff
@@ -301,3 +302,26 @@ class JobCache(Base, Timestamp):
     result_path = Column(String(1024))
     workspace_hash = Column(String(1024))
     access_times = Column(JSONType)
+
+
+class AuditLogAction(enum.Enum):
+    """Enumeration of audit log actions."""
+
+    request_token = 0
+
+
+class AuditLog(Base, Timestamp):
+    """Audit log table."""
+
+    __tablename__ = 'audit_log'
+
+    id_ = Column(UUIDType, unique=True, primary_key=True,
+                 default=generate_uuid)
+    user_id = Column(UUIDType, ForeignKey('user_.id_'), nullable=False)
+    # created_at = Column(DateTime) # comes from Timestamp (created)
+    # modify_at = Column(DateTime) # comes from Timestamp (updated)
```

Comment:
Just for you to know that I didn't create these columns because apparently they come with `Timestamp`.